src/component/plotly-dash-preview/render.js

```diff
@@ -0,0 +1,65 @@
+const remote = require('../../util/remote')
+
+/**
+ * @param {object} info : info object
+ *  - url
+ *  - pdfOptions
+ * @param {object} opts : component options
+ * @param {function} sendToMain
+ *  - errorCode
+ *  - result
+ *    - imgData
+ */
+function render (info, opts, sendToMain) {
+  const result = {}
+
+  let win = remote.createBrowserWindow()
```

Comment:
No need to specify the window width/height? That is, you're assuming all dash reports will have the same dimensions?

Reply:
Some discussion on this earlier here: https://github.com/plotly/image-exporter/pull/60#discussion_r174827073

Now that I think about it, I wonder if we should still fix the size of the window before specifying the size of the PDF export. The PDF Export size will likely change the viewport (so responsive CSS will be reflected in the PDF) but I doubt that it will re-renderer all of the javascript. So, if there are certain components that are responsive with JS and not CSS, then I doubt that they will be reflected at the new size.

So, I think it makes sense to set size of this window to be the same size as requested in the PDF renderer. For printed content, 1px will be 1/96th of an inch (https://www.w3.org/Style/Examples/007/units.en.html). So 8.5 x 11 is 816px x 1056

Reply:
Elaborating on ⬆️ my point was that:

> A4 paper measures in at 210mm by 297mm (8.26" by 11.69") whereas US Letter paper measures at 8.5" by 11" (215.9mm by 279.4mm)

Since difference is not too much between A4 and Letter, it shouldn't really affect the viewports if we change it by the `paperSize` (though we do get a significant difference when we consider A3/A2 as well)

If we consider A2, it comes out to ~`1584.0px x 2246px` .. which is big compared to general screen-sizes. (or `xvfb` screen sizes in case of headless server)